### PR TITLE
Python3 port for test_process.py

### PR DESCRIPTION
**Pull request by kdcis**
- Enabled Python3

- Fix warning: Unclosed File Error

### CODE REVIEW

File: src/tests/test_process.py [hunk 1]
```diff
@@ -87,7 +84,8 @@ def setUp(self):
     def _get_readline(cls, pfile):
         pfile = os.path.join(cls.home, pfile)
         try:
-            return open(pfile, 'rb').readline().strip()
```
Comment (PeterSurda) [inline src/tests/test_process.py:90]: good

File: src/pybitmessage [hunk 2]
```diff
@@ -8,4 +8,7 @@ script_file = os.path.join(dist.location, dist.key, 'bitmessagemain.py')
 new_globals = globals()
 new_globals.update(__file__=script_file)
 
-execfile(script_file, new_globals)
```
Comment (PeterSurda) [inline src/pybitmessage:11]: This is probably ok, but other parts of the code refuse to run in python3, or may be not compatible. If you remove the refusals, you'll get more errors and those need to be fixed. However, maybe it makes sense to fix those first before allowing this script to run in python3 as that may be a lot of work.

**Review by PeterSurda — CHANGES_REQUESTED** [review]

**Comment by g1itch** [comment]
It's a bad idea to edit this workaround artifact. The main problem is the circular import in the `network` package and not sorted side effects of such imports as `debug`, `helper_startup`. That all issues are preventing us from using the right script kwarg:  https://github.com/Bitmessage/PyBitmessage/blob/v0.6/setup.py#L171

And also you've already informed about `TestProcessProto` and the blind tests: they are going to be finished last because are using the whole pybitmessage daemon installed in the system.

**Comment by g1itch** [comment]
The main idea of the fake script was to use `pathmagic` in setuptools generated script, or rather something close enough to it, not to reinvent the wheel, but this particular one is taken from gentoo `python-exec` product and it was a miracle that it works everywhere.

**Comment by PeterSurda** [comment]
Only keep the bottom most change, revert everything else.

**Comment by kdcis** [comment]
> Only keep the bottom most change, revert everything else.

Updated.

**Review by PeterSurda — CHANGES_REQUESTED** [review]
Remove first hunk, only keep second hunk.

**Comment by kdcis** [comment]
> Remove first hunk, only keep second hunk.

Updated accordingly.

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**